src/stories/Filters.js

```diff
@@ -35,71 +33,68 @@ const Tag = ({ tag }) => (
 )
 
 const setViewsAction = action('Set views')
-const setRulesAction = action('Set rules')
-
-const DateTimeDisplay = PineTypes['Date Time'].Display
+const setFiltersAction = action('Set filters')
 
 const schema = {
-  Category: {
-    type: 'Enum',
-    values: uniq(PokeDex.map(p => p.Category))
-  },
-  Name: {
-    type: 'Short Text'
-  },
-  Description: {
-    type: 'Short Text'
-  },
-  Abilities: {
-    type: 'Short Text'
-  },
-  Height: {
-    type: 'Real'
-  },
-  Weight: {
-    type: 'Real'
-  },
-  first_seen: {
-    type: 'Date Time',
-    label: 'First Seen'
-  },
-  pokedex_number: {
-    type: 'Integer',
-    label: 'National Pokedex Number'
-  },
-  caught: {
-    type: 'Boolean',
-    label: 'Pokemon has been caught'
-  },
-  Tag: {
-    type: 'Key Value Pair',
-    key: 'tag_name',
-    value: 'tag_value',
-    keyLabel: 'Name',
-    valueLabel: 'Value'
+  type: 'object',
+  properties: {
+    Name: {
+      title: 'Pokemon Name',
+      type: 'string'
+    },
+    Description: {
+      title: 'Pokemon Name',
+      type: 'string'
+    },
+    Abilities: {
+      title: 'Pokemon Name',
```

Comment:
I think that these titles should be changed.
![image](https://user-images.githubusercontent.com/1295829/38245775-2ae236d0-3748-11e8-87c5-3a0d58266989.png)